### ISSUE TITLE
experimental/Firmware/esp32_display.bin build 2111281

### ISSUE THREAD

**Issue by o1hitman1o** [srg74/WLED-wemos-shield#61]
experimental/Firmware/esp32_display.bin The display is not functioning. But in experimental/Firmware/Debug_builds/esp32_display.bin Everything is working correctly.
Not sure what the difference is.

**Comment by srg74** [comment]
There is no difference in builds, only debug build is debug enabled.

**Comment by o1hitman1o** [comment]
I have tested every build from v0.11.1_210210 on v1.51 board to the latest version. This was the results.

wemos_shield_esp8266.bin v0.11.1_2102210 - working with display
wemos_shield_esp8266.bin v0.11.1_2102130 - working with display
wemos_shield_esp8266_display.bin v0.11.1_2102130 Update to recent - working no display
esp8266_display.bin v0.12.0_2106280 working no display
esp8266_display.bin v0.13.0-b2 - working no display
esp8266_display.bin v0.13.0-b3 - working no display
esp8266_display.bin v0.13.0-b4 - won't boot
esp8266_display.bin v0.13.0-b5 - won't boot
esp8266_display.bin experimental - working no display
esp8266_display.bin experimental debug - working with display
 I have tried flashing several times and get the same result.
There has to be some difference somewhere I just don't know where to look.
